permissions/rolls/permissions.ps1

```diff
@@ -0,0 +1,149 @@
+#################################################
+# HelloID-Conn-Prov-Target-Eduarte-Medewerker-Permissions-Rolls
```

Comment:
Je hebt het hier over Rolls, maar dit zou in mijn optiek Roles moeten zijn. Als jij hier in mee gaat, zou het het overal moeten aanpassen. 

Wel een nice to have. maar nu leest het wat raar.